src/test/java/core/basesyntax/operations/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.operations;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.storage.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    private static PurchaseOperationHandler operationHandler;
+    private static Fruit apple;
+    private static Fruit banana;
+
+    @BeforeClass
+    public static void beforeClass() {

```

Comment:
```suggestion
    public static void init() {
```
the same for other similar cases